src/points/PointsHistoryBottomSheet.tsx

```diff
@@ -55,6 +57,8 @@ function PointsHistoryBottomSheet({ forwardedRef }: Props) {
   const pointsHistoryStatus = useSelector(pointsHistoryStatusSelector)
   const pointsHistory = useSelector(pointsHistorySelector)
 
+  const [showErrorBanner, setShowErrorBanner] = useState(false)
```

Comment:
```suggestion
  const [showError, setShowError] = useState(false)
```
nit: technically not a banner (and the type of display component could change)